gcp-repo/metadata.p2.inf

```diff
@@ -36,5 +36,5 @@ Please visit the following URL for details:\n\n\
 units.1.licenses.0.location = https://www.apache.org/licenses/LICENSE-2.0
 units.1.requires.1.namespace=org.eclipse.equinox.p2.iu
 units.1.requires.1.name=com.google.cloud.tools.eclipse.suite.e45.feature.feature.group
-units.1.requires.1.range=[1.0.0,1.0.0]
+units.1.requires.1.range=[1.0.0,1.0.1)
```

Comment:
Or you could drop this line entirely.  The danger is if you have a version in your local repository, but that can be avoided by setting the `tycho.localArtifacts` property (as per the _travis_ profile).  We probably should set up a _release_ profile.